schemas/documents/terrasos/tebuBanner.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  name: 'tebuBanner',
+  type: 'document',
+  title: 'Tebu Banner',
+  fields: [
```

Comment:
I'm wondering if we want the tebu logo to be on sanity too or not, cc/ @erikalogie see https://www.figma.com/design/brkxGV5qNOkZUp0YQl1cxO/Terrasos-Phase-1?node-id=1111-319296&node-type=rounded_rectangle&m=dev

Reply:
I'll make everything configurable, just in case.